src/aics-image-viewer/components/ControlPanel/index.jsx

```diff
@@ -223,11 +224,11 @@ export default class ControlPanel extends React.Component {
           ]}
         />
         <Card.Meta
-          title={[
-            this.renderAxesButton(),
-            this.renderBoundingBoxButton(),
-            this.renderResetCameraButton()
-          ]}
+          title={<>
+            {this.renderAxesButton()}
+            {this.renderBoundingBoxButton()}
+            {this.renderResetCameraButton()}
+          </>}
```

Comment:
what's the reason for this change?

Reply:
React was complaining that this array of components did not have indexes; fragments don't have that requirement. I'm pretty sure this isn't the best practice (I believe fragments are meant to be used pretty much exclusively as the top-level container in JSX, to allow a component's `render` function to return multiple elements) but these buttons are soon to be moved out of the control panel anyways and into the new toolbar.